### ISSUE TITLE
Empty lang attribute on Download button on From detail pages

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#20079]
## Description
Martha sent this over rom the latest AMP scans:

On the Download buttons on form detail pages, there is an empty `lang` attribute and it's being flagged in AMP. This is the tool VA looks at to determine their accessibility so it would be good to get this cleaned up.

[Example form page](https://www.va.gov/find-forms/about-form-10-493a/)

![Image](https://github.com/user-attachments/assets/49883f50-535d-4932-96f3-87e7e8670fcd)

### Scope / Impact analysis
_What, if anything, could break as a result of this change?_
_Engineer should assess this when approaching PR._

### Roles / assignments
_After functional testing, code review, accessibility review, and design review can happen in parallel._
- [ ] **Functional testing:** 
- [ ] **Code review:** 
- [ ] **Design review:** 
- [ ] **Accessibility review:** 
- [ ] **Product acceptance:** 

## Acceptance criteria
_ACs should capture expected behavior, to inform test plans. Consider devices, documentation updates including KBs, change management, and [content model](https://prod.cms.va.gov/admin/structure/cm_document) when applicable._

- [ ] Download buttons do not have a 'lang' attribute (We don't have spanish pages currently so I don't think we need this?
- [ ] Impact analysis is filled out and QA test plan is updated accordingly
- [ ] Does this product have an existing regression test plan?
	- [ ] Yes, link: 
	- [ ] No, regression test plan will be created as part of the work / points in this ticket

**Comment by FranECross** [comment]
@laflannery Just to confirm, this ticket is just to correct the 'lang' element for the download buttons specifically on the two form detail pages you provided?

**Comment by laflannery** [comment]
No, those were examples, I had put the necessary AC in the ticket originally:

- [ ] Download buttons do not have an empty 'lang' attribute

**Comment by jilladams** [comment]
@artsymartha68 @laflannery is the Amp report comprehensive of all VA.gov forms pages? If so, could we get a copy that would show us how many pages have this issue? 


From refinement: 
* Forms DB data does not always include the language on a Form. We can't control that, so sometimes we will not have the data to properly populate this attribute.
* Shamia's take: it's better to have an empty lang attribute than no lang attribute at all.
* We don't know the scale of how many forms don't have an assigned language. That data is in Drupal, but not exposed yet. 
   * If Laura / Martha can provide us the AMP report and it's comprehensive, we can tabulate which forms fail this way.
   * If not: we can expose the Language option on the VA Forms drupal audit view (https://prod.cms.va.gov/admin/content/va-forms/audit), and get that data there. (I'd actually prefer this route, cuz it would be useful for future. I'm gonna stub the ticket as a nice to have.)
    * Once we know how many forms are affected, we can reach out to Forms Managers with the list of forms where language isn't populated, to ask them to do so. (This would be an opportunity to also talk to them about standardizing how Spanish / English forms are indicated, in service of #19621 )

**Comment by jilladams** [comment]
Ok - I went ahead and stubbed a ticket to add a filter on the Forms audit view: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/20358. Christian took a look and said it's a very light lift, so that should help.

**Comment by jilladams** [comment]
#20358 will add a language filter to the form audit view - that'll be available after the next successful CMS deploy. Once deployed, these URLs will show the count of forms where the language is declared (right now, the filter doesn't exist yet, so you'll get a list of all forms instead):

* [English](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=All&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=en)
* [Spanish](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=All&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=es)
* [Tagalog](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=All&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=tl)
* [No lang declared](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=All&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=empty&field_va_form_language_value=tl)

To get a valid count of forms by language, we'll need to count Published + Draft (and exclude Archived).

**Comment by jilladams** [comment]
Audit data now that we have the view filter: 


689 total forms in Draft + Published

Lang | Published | Draft | Total
--- | --- | --- | ---
English | [165](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=en) | [279](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=draft&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=en) | 444 total
Spanish | [0? blank view](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=es) |  [2](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=draft&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=es) | 2
Tagalog | [0?](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=tl) | 0 | 0
No language declared | [241 total](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=empty&field_va_form_language_value=tl) <br/>[126 VBA](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=1&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=empty&field_va_form_language_value=tl) <br/>[112 VHA](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=3&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=empty&field_va_form_language_value=tl) <br/>3 VACO | [2](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=draft&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=empty&field_va_form_language_value=tl) both VHA | 243 total

**Comment by jilladams** [comment]
@mmiddaugh @FranECross Now that we can filter the Forms audit view by language, we have an initial count / rundown of which forms need language added. Let me know if you'd like me to help with any next steps with forms managers.

689 total forms in Draft + Published

Lang | Published | Draft | Total
--- | --- | --- | ---
English | [165](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=en) | [279](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=draft&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=en) | 444 total
Spanish | [0? blank view](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=es) |  [3](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=draft&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=es) | 2
Tagalog | [0?](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=or&field_va_form_language_value=tl) | 0 | 0
No language declared | [241 total](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=empty&field_va_form_language_value=tl) <br/>[126 VBA](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=1&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=empty&field_va_form_language_value=tl) <br/>[112 VHA](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=published&workbench_access_section__section=All&field_va_form_administration_target_id=3&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=empty&field_va_form_language_value=tl) <br/>3 VACO | [2](https://prod.cms.va.gov/admin/content/va-forms/audit?combine=&moderation_state=draft&workbench_access_section__section=All&field_va_form_administration_target_id=All&field_va_form_issue_date_value_op=%3E%3D&field_va_form_issue_date_value%5Bvalue%5D=&field_va_form_issue_date_value%5Bmin%5D=&field_va_form_issue_date_value%5Bmax%5D=&field_va_form_revision_date_value_op=%3E%3D&field_va_form_revision_date_value%5Bvalue%5D=&field_va_form_revision_date_value%5Bmin%5D=&field_va_form_revision_date_value%5Bmax%5D=&field_va_form_usage_value_op=contains&field_va_form_usage_value=&field_va_form_tool_url_uri_op=contains&field_va_form_tool_url_uri=&field_va_form_language_value_op=empty&field_va_form_language_value=tl) both VHA | 243 total

**Comment by FranECross** [comment]
@jilladams It seems the only way to suss out what language a form is will be to look at each one, and then add the language in CMS,? Do I have that correct? If so, I can create a ticket for myself to audit (which will be little by little.

**Comment by jilladams** [comment]
Yeah, I hadn't really done the math but there doesn't appear to be a Language field in the Forms DB, or if there is we aren't receiving it and aren't mapping it in the migration. I may poke Kevin Reid about that just to make sure. If they have a field we aren't receiving, we could receive it and make managers use it. But if not, Forms DB is in maintenance mode and they're unlikely to add that I think, so yeah: we'd have to do all the data updates manually in Drupal.
